conformance/utils/suite/suite.go

```diff
@@ -77,6 +77,23 @@ var StandardCoreFeatures = map[SupportedFeature]bool{
 	SupportReferenceGrant: true,
 }
 
+// AllFeatures contains all the supported features and can be used to run all
+// conformance tests with `all-features` flag.
+var AllFeatures = map[SupportedFeature]bool{
```

Comment:
is there any way a contributor defines the suite name once ? I fear the contributor will add it the constant w/o adding it to this map, and the burden to ensure its in sync will fall on the maintainer

Reply:
Yes, you are right, developers need to modify two places after the constant changes. But I can't find a simple way to define the suite name once in this codebase. Or I can add a comment to notice developer/code reviewer to ensure they are in sync?
@robscott Can you help with some advice?

Reply:
I can't think of a better solution rn, can you add a comment to ensure they are in sync, TIA !